<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Amine say</author>
 <class>smshttpClass</class>
 <widget class="QMainWindow" name="smshttpClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>556</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>smshttp</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/resources.qrc">
    <normaloff>:/icons/icon32.ico</normaloff>:/icons/icon32.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="logBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Number   :</string>
           </property>
           <property name="buddy">
            <cstring>userPhone</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="userPhone">
           <property name="toolTip">
            <string>Your smshttp login</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Password :</string>
           </property>
           <property name="buddy">
            <cstring>userPass</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="userPass">
           <property name="toolTip">
            <string>Your Smshttp password</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pb_logIn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Log &amp;In</string>
         </property>
         <property name="icon">
          <iconset resource="resources/resources.qrc">
           <normaloff>:/icons/dialog-ok-apply.png</normaloff>:/icons/dialog-ok-apply.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_rememberMe">
         <property name="text">
          <string>&amp;Remember Me</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_autoLogin">
         <property name="text">
          <string>&amp;Login automatically</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pb_help">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>&amp;Help</string>
           </property>
           <property name="icon">
            <iconset resource="resources/resources.qrc">
             <normaloff>:/icons/help-about.png</normaloff>:/icons/help-about.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="6">
     <widget class="QGroupBox" name="smsBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTabWidget" name="tw_contactsGroups">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>130</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>135</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_contacts">
          <attribute name="title">
           <string>Contacts</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_6">
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item row="0" column="0" rowspan="3">
            <layout class="QGridLayout" name="gridLayout_5">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>Name:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="2">
              <widget class="QLineEdit" name="nameHint">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>Contacts:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1" colspan="2">
              <widget class="QComboBox" name="cb_contacts">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Use the contact list to automatically fill the phone number</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="4" column="0">
            <widget class="QPushButton" name="contactsAdd">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>26</height>
              </size>
             </property>
             <property name="text">
              <string>&amp;Add</string>
             </property>
             <property name="icon">
              <iconset resource="resources/resources.qrc">
               <normaloff>:/icons/list-add.png</normaloff>:/icons/list-add.png</iconset>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_groups">
          <attribute name="title">
           <string>Groups</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_7">
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item row="2" column="0" colspan="2">
            <widget class="QPushButton" name="groupsAdd">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>26</height>
              </size>
             </property>
             <property name="text">
              <string>&amp;Add</string>
             </property>
             <property name="icon">
              <iconset resource="resources/resources.qrc">
               <normaloff>:/icons/list-add.png</normaloff>:/icons/list-add.png</iconset>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>Groups :</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="cb_groups">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Use a group to send a SMS to multiple persons at once</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_phone">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <attribute name="title">
           <string>Phone</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_8">
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_4">
             <property name="sizeConstraint">
              <enum>QLayout::SetMaximumSize</enum>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_6">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>Phone:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="contactPhone">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Phone number to use to send SMS</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="phoneAdd">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>26</height>
              </size>
             </property>
             <property name="text">
              <string>&amp;Add</string>
             </property>
             <property name="icon">
              <iconset resource="resources/resources.qrc">
               <normaloff>:/icons/list-add.png</normaloff>:/icons/list-add.png</iconset>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="sendList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="toolTip">
          <string>List of contact to send the SMS to</string>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="smsText">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>30</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>75</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>300</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Type your SMS here</string>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="backgroundVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_charLeft">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="pb_clear">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Cl&amp;ear</string>
           </property>
           <property name="icon">
            <iconset resource="resources/resources.qrc">
             <normaloff>:/icons/edit-clear.png</normaloff>:/icons/edit-clear.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_send">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&amp;Send</string>
           </property>
           <property name="icon">
            <iconset resource="resources/resources.qrc">
             <normaloff>:/icons/mail-mark-unread.png</normaloff>:/icons/mail-mark-unread.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_validation">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Ask &amp;validation before sending</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_confirmation">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Wait for delivery confirmation</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QGroupBox" name="creditsBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="creditsLeft">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>54</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="lb_creditsLeft">
           <property name="text">
            <string>Credits left</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pb_manageContacts">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Manage &amp;Contacts</string>
      </property>
      <property name="icon">
       <iconset resource="resources/resources.qrc">
        <normaloff>:/icons/list-add-user.png</normaloff>:/icons/list-add-user.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>23</width>
        <height>23</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pb_manageMessages">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Manage &amp;Messages</string>
      </property>
      <property name="icon">
       <iconset resource="resources/resources.qrc">
        <normaloff>:/icons/mail-queue.png</normaloff>:/icons/mail-queue.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>userPhone</tabstop>
  <tabstop>userPass</tabstop>
  <tabstop>pb_logIn</tabstop>
  <tabstop>cb_rememberMe</tabstop>
  <tabstop>cb_autoLogin</tabstop>
  <tabstop>pb_help</tabstop>
  <tabstop>pb_manageContacts</tabstop>
  <tabstop>pb_manageMessages</tabstop>
  <tabstop>tw_contactsGroups</tabstop>
  <tabstop>nameHint</tabstop>
  <tabstop>cb_contacts</tabstop>
  <tabstop>contactsAdd</tabstop>
  <tabstop>sendList</tabstop>
  <tabstop>smsText</tabstop>
  <tabstop>pb_send</tabstop>
  <tabstop>pb_clear</tabstop>
  <tabstop>cb_validation</tabstop>
  <tabstop>cb_confirmation</tabstop>
  <tabstop>cb_groups</tabstop>
  <tabstop>groupsAdd</tabstop>
  <tabstop>contactPhone</tabstop>
  <tabstop>phoneAdd</tabstop>
 </tabstops>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
